File: backend/src/main/java/com/bv87/diemdanh/dto/ManualSchedulePeriodDto.java
package com.bv87.diemdanh.dto;

import lombok.Builder;
import lombok.Getter;

import java.time.LocalDate;

/** One merged manual-status period — SPEC §3.2.2. */
@Getter
@Builder
public class ManualSchedulePeriodDto {
    private final LocalDate fromDate;
    private final LocalDate toDate;
    private final int dayCount;
    private final String status;
    private final String statusLabel;
}

File: desktop/src/BV87.Core/Models/Attendance/ManualSchedulePeriodDto.cs
namespace BV87.Core.Models.Attendance;

public sealed class ManualSchedulePeriodDto
{
    public DateOnly FromDate { get; set; }
    public DateOnly ToDate { get; set; }
    public int DayCount { get; set; }
    public string? Status { get; set; }
    public string? StatusLabel { get; set; }
}
